src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/esa/EsaWriteFinalDecisionPreviewDecisionService.java

```diff
@@ -141,7 +145,8 @@ protected void setDescriptorsAndPoints(WriteFinalDecisionTemplateBodyBuilder bui
             .stream().map(q ->
                 buildDescriptorFromActivityQuestion(activityQuestionlookup.getByKey(q))).collect(Collectors.toList());
 
-        descriptors.sort(new DescriptorLexicographicalComparator());
+        // FIXME Extract sort order from question text
```

Comment:
Will this be fixed as part of separate PR?

Reply:
Yes, as discussed